src/atoms/map/LeafletMap/LeafletMap.vue

```diff
@@ -18,14 +18,16 @@ const props = withDefaults(
     customControls?: Control[]
     contextMenu?: boolean
     hideControls?: boolean
+    osmBaseMap?: boolean
   }>(),
   {
     minZoom: 0,
     maxZoom: 19,
     initialZoom: 14,
     customControls: () => [],
     contextMenu: false,
-    hideControls: false
+    hideControls: false,
+    osmBaseMap: true
```

Comment:
Maybe we can introduce a ref that contains one or multiple base layers. So it is possible to actively choose the base layer. If no base layer was given the default is used.

Reply:
Maybe it is better to have a flag. For digimap we need a list and a lot more logic and with a single layer we cannot define that we do not want any layer. If the ref is not set we would display the base layer. So this approach should be fine for now.